Makefile

```diff
@@ -281,6 +285,26 @@ push:
 rsh:
 	oc exec -it cas-ggircs-5-c46gh -- bash
 
+.PHONY: refresh
```

Comment:
should we have `refresh_test` and `refresh_dev`? Also, maybe `refresh_views` is a better name for it?

Reply:
yes, it was an example use of [gnu parallel](https://www.gnu.org/software/parallel/) that I don't think we'll be moving ahead with

Reply:
commented out for now, in case we need it